main.cpp

```diff
@@ -213,7 +231,21 @@ void Band::postOnInsta(int numOfPics, char filter, std::string postText)
     numOfPics = 5;
     filter = 'N';
     postText = "Had an awesome time at Club Buddhabump last nite!";
-    std::cout << postText << std::endl;;
+    std::cout << "Insta text: " << postText << std::endl;
+}
+
+double Band::goOnTour(int numOfDestinations, double audience, bool isGrowing)
+{
+    numOfSongs = 15;
+    budget = 5000.0;
+    for(int i = 0 ; i < numOfDestinations ; ++i )
+    {
+        (isGrowing) ? audience *= 2.22 : audience /= 2.22;
```

Comment:
This is a bad coding style to develop. 
you should either use `if( isGrowing ) ...`
or
```
audience = (isGrowing) ? audience * 2.22 : audience / 2.22;
```
Always strive to make your code as easy to understand as possible
- [ ] revise